### ISSUE TITLE
It is not possible to read Timeline content after selecting one of the tabs using Firefox/NVDA

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4383]
## Describe the bug

[Test URL]
https://clarity.design/documentation/timeline

[WCAG 2.1 Guideline]
Success Criterion 1.3.2 Meaningful Sequence
(Level A)
https://www.w3.org/TR/WCAG21/#meaningful-sequence

[OS]
Windows 10

[User Agents/Assistive Technologies]
Firefox/NVDA

[Component]
Timeline

[Description]
It is not possible to read Timeline content after selecting one of the tabs using Firefox/NVDA.

[User Impact]
Firefox/NVDA users will not be able to read the content associated with the selected tab.

[Steps to Reproduce]
- Go the Test URL
- Using a Firefox/NVDA select one of the tabs in the "Full Demo" section of the page
- Try to read the tabpanel content using the down arrow key and observe you cannot

[Recommendation]
Ensure that the tabs implementation matches the guidance at https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html

**Comment by gnomeontherun** [comment]
This appears to just be a website issue, or else all of our tabs have this problem. This will likely be removed in upcoming website work, marking as low.
